com.unity.multiplayer.mlapi/Runtime/Profiling/ProfilerCountersInfo.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+#if UNITY_2020_2_OR_NEWER
+using Unity.Profiling;
+#endif
+using UnityEngine;
+
+namespace MLAPI.Profiling
+{
+
+    public static class ProfilerCountersInfo
+    {
+#if UNITY_2020_2_OR_NEWER && ENABLE_PROFILER
+        // Operations
+        public static ProfilerCounterValue<int> ConnectionsCounterValue =
```

Comment:
Nit, all these counters can be private I think. With them being public we are exposing them to be modified, which isn't the intent

Reply:
Agreed. Updated